src/main/java/frc/robot/subsystems/Chamber.java

```diff
@@ -136,20 +138,37 @@ public boolean shootNote(double leftSpeed, double rightSpeed) {
     }
   }
 
+  public boolean isNoteAligned() {
+    if (isNoteDetected() && !m_noteUnderSensor) {
+      m_counter++;
+      m_noteUnderSensor = true;
+    } else if (!isNoteDetected()) {
+      m_noteUnderSensor = false;
+    }
+    if (m_counter == 2 && !m_noteUnderSensor) {
+      return true;
+    }
+    return false;
+  }
+
   @Override
   public void periodic() {
     switch (m_state) {
       case EMPTY: {}
       case CHAMBERING: {
         if (isNoteDetected()) {
           m_intake.stopIntake();
-          stopChamber(); // TODO: Change motor speeds for aligning.
+          setSpeed(ChamberConstants.kAligningSpeed);
           m_state = State.ALIGNING;
         }
       }
       case ALIGNING: {
-        // TODO: Align note.
-        // TODO: Once aligned, stop chamber motors, idle shooter, change state to CHAMBERED.
+        if (isNoteAligned()) {
+          stopChamber();
+          m_shooter.idleShooter();
+          m_state = State.CHAMBERED;
+        }
+
```

Comment:
It looks like this pull request is based on an old version that was missing `break` statements. Make sure the one for this block doesn't get lost during rebasing.